aristamd/src/app/components-section/accordion-demo/accordion-demo.component.js

```diff
@@ -0,0 +1,50 @@
+class AccordionDemoController {
+  constructor($scope, $uibModal, $timeout, communicationCenterService) {
+    const ctrl = this;
+
+    ctrl.$onInit = function () {
```

Comment:
Have you tried to declare $onInit as a class method instead of a property inside constructor?